pkg/logging/logging.go

```diff
@@ -0,0 +1,103 @@
+// package logging is a small wrapper around github.com/k8snetworkplumbingwg/cni-log
+
+package logging
+
+import (
+	cnilog "github.com/k8snetworkplumbingwg/cni-log"
+)
+
+const (
+	labelCNIName     = "cniName"
+	labelContainerID = "containerID"
+	labelNetNS       = "netns"
+	labelIFName      = "ifname"
+	cniName          = "sriov-cni"
+)
+
+var (
+	logLevelDefault = cnilog.ErrorLevel
+	containerID     = ""
+	netNS           = ""
+	ifName          = ""
+)
+
+func init() {
+	SetLogFile("")
+	SetLogLevel("")
```

Comment:
init of cni-log pkg calls initloger which already sets empty file and enalbe stderr. So, no need to have it here again. If you decide to set error as log level (instead of info as set by cni-log, see comment above). I think this should be the place.

Reply:
I removed init() altogether